src/assets/images/follow.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="Component_2_1" data-name="Component 2 – 1" xmlns="http://www.w3.org/2000/svg" width="78.06" height="23.995" viewBox="0 0 78.06 23.995">
+  <path id="ic_add_circle_24px" d="M14,2A12,12,0,1,0,26,14,12,12,0,0,0,14,2Zm6,13.2H15.2V20H12.8V15.2H8V12.8h4.8V8h2.4v4.8H20Z" transform="translate(52.065 -2)" fill="#0dc68d"/>
+  <text id="Follow" transform="translate(22 17.318)" fill="#0dc68d" font-size="15" font-family="Helvetica"><tspan x="-21.672" y="0">Follow</tspan></text>
+</svg>
```

Comment:
Please add a blank line at the end of this file

Reply:
It is now done.